artemis-server/src/test/java/org/apache/activemq/artemis/core/config/impl/FileConfigurationParserTest.java

```diff
@@ -775,6 +776,7 @@ public void testMaxSize() throws Exception {
 
       assertEquals(10 * 1024 * 1024, configuration.getGlobalMaxSize());
       assertEquals(1000, configuration.getGlobalMaxMessages());
+      assertEquals(30, configuration.getGlobalMaxSizePercentOfJvmMaxMemory());
```

Comment:
I would use a different number from the one you used on the XML.. in case the XML gets parsed by accident.